packages/mermaid/src/utils.ts

```diff
@@ -838,6 +838,32 @@ export function getErrorMessage(error: unknown): string {
   return String(error);
 }
 
+/**
+ * Appends <text> element with the given title, centered.
+ *
+ * @param {any} parent d3 svg object to append title to
+ * @param {string} cssClass
+ * @param {number} titleTopMargin
+ * @param {string | undefined} title
```

Comment:
Thanks for making those changes :)

One last thing, not 100% required, but can you add a quick description for these parameters?

Otherwise, we might get some JSDoc ESLint warnings (or we might get some warnings in the future).

Reply:
Actually, now that https://github.com/mermaid-js/mermaid/pull/3708 is merged, you'll definitely need to change these.

TypeScript `.ts` files now need TSDoc comments, not JSDoc comments (you may need to update your PR from the latest changes from `develop`)

Basically, you'll have to remove the type (since it's already in the function), and add a `-` before the description, e.g.

```ts
 * @param parent - d3 svg object to append title to
 * @param cssClass - CSS class to give the `<title>` element
 * @param titleTopMargin - Pixels between title and top of SVG
 * @param title - Title contents (if blank, this does nothing)
```

Reply:
Okay, done in ccd88c8b2928f09823e7838216f93335f7a496dc